lib/modules/apostrophe-ui/public/js/ui.js

```diff
@@ -18,13 +18,64 @@ apos.define('apostrophe-ui', {
       self.enableActionable();
       self.enhanceDropdowns();
       self.enableAjax();
+      self.enableTooltips();
     });
   },
 
   construct: function(self, options) {
 
     self.options = options;
 
+    // Utility that lets you add a tooltip to any HTML element using
+    // the data-apos-tooltip="YR TOOLTIP MSG" pattern
+
+    self.enableTooltips = function() {
+      var $tooltip = $('[data-apos-tooltip-template]');
+      var $clone = $tooltip.clone();
+      var $body = $('body');
+      $clone
+        .removeAttr('data-apos-tooltip-template')
+        .attr('data-apos-tooltip', 'true')
+        .removeClass('apos-hidden');
+
+      $('[data-apos-tooltip]').hover(function() {
```

Comment:
This also needs to be fixed, yes, to use jquery event delegation as Austin suggests.